A/armadillo/build_tarballs.jl

```diff
@@ -6,8 +6,8 @@ using BinaryBuilder
 name = "armadillo"
 version = v"9.850.1"
 sources = [
-    ArchiveSource("http://sourceforge.net/projects/arma/files/armadillo-9.850.1.tar.xz",
-                  "d4c389b9597a5731500ad7a2656c11a6031757aaaadbcafdea5cc8ac0fd2c01f")
+    "http://sourceforge.net/projects/arma/files/armadillo-9.850.1.tar.xz" =>
```

Comment:
?

Reply:
Ack, this was a local change that wasn't supposed to be checked in.  My local Julia version does not work with `ArchiveSource` (something is out of date but I haven't bothered to figure out what).

Reply:
`ArchiveSource` is only on `master` of `BinaryBuilder.jl`.  You know, we love to live on the bleeding edge :stuck_out_tongue:

Reply:
I figured it was something like that. :)